docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+
+            /// Unset the callback
+            VoiceControlManagerClient.StateChanged -= stateChanged;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the service state changed callback to be invoked when the voice control manager service state changes:
+    ```
+    /// callback
+    void serviceStateChanged(object sender, ServiceStateChangedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetServiceStateChangedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.ServiceStateChanged += serviceStateChanged;
+
+            /// Unset the callback
+            VoiceControlManagerClient.ServiceStateChanged -= serviceStateChanged;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the current language changed callback invoked when the voice control settings language changes:
+    ```
+    /// callback
+    void currentLanguageChanged(object sender, CurrentLanguageChangedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetCurrentLanguageChangedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.CurrentLanguageChanged += currentLanguageChanged;
+
+            /// Unset the callback
+            VoiceControlManagerClient.CurrentLanguageChanged -= currentLanguageChanged;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the error callback to be invoked when an error occurs in the voice control manager process:
+    ```
+    /// callback
+    void errorOccurred(object sender, ErrorOccurredEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetErrorOccurredCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.ErrorOccurred += errorOccurred;
+
+            /// Unset the callback
+            VoiceControlManagerClient.ErrorOccurred -= errorOccurred;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the speech detected callback invoked when beginning of speech or end of speech is detected:
+    ```
+    /// callback
+    void speechDetected(object sender, EventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetSpeechDetectedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SpeechDetected += speechDetected;
+
+            /// Unset the callback
+            VoiceControlManagerClient.SpeechDetected -= speechDetected;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the conversation requested callback invoked when conversation requests in the speech control manager process:
```

Comment:
@sungraejo Would you please elaborate the sentence so that I or @rishu-samsung can check?

Reply:
Changed


-   Set the conversation requested callback invoked when conversation requests in the speech control manager process:
to
-   Set the conversation requested callback invoked when the voice control manager or engine requests conversation for additional information about current utterence: